Sources/SwiftGen/Config/ConfigOutput.swift

```diff
@@ -52,21 +52,23 @@ extension ConfigEntryOutput {
 /// Convert to CommandLine-equivalent string (for verbose mode, printing linting info, …)
 ///
 extension ConfigEntryOutput {
-  func commandLine(forCommand cmd: String, inputs: [Path]) -> String {
+  func commandLine(forCommand cmd: String, inputs: [Path], filter: String?) -> String {
     let tplFlag: String = {
```

Comment:
Occasion to rename `s/tplFlag/templateFlag/`? That short name just felt strange when re-reading this code…